README.md

```diff
@@ -1,5 +1,7 @@
 ![Siembol](logo.svg)
 
+![Black Hat Arsenal](https://raw.githubusercontent.com/toolswatch/badges/54ad78bc63b24ce445e8241f179fe1ddeecf8eef/arsenal/usa/2021.svg)
```

Comment:
@c-rindi  Thanks for PR. LGTM, maybe we can add link to the presentation (https://www.blackhat.com/us-21/arsenal/schedule/index.html#siembol-an-open-source-real-time-siem-tool-based-on-big-data-technologies-24038) ?

Reply:
Good idea - link added.